src/main/java/org/session4/homework/Person.java

```diff
@@ -4,11 +4,11 @@ public class Person {
 
     int age;
 
-    public Person(int age){
+    public Person(int age) {
         this.age = age;
     }
 
-    public String displayIfThePersonIsAdult(){
+    public String adulthoodOfPerson() {
         return age >= 18 ? "I am adult." : "I am not adult.";
```

Comment:
Does this method actually respond to yes or no questions? If yes, I suggest trying to delegate the responsibilities, here we can just check if the person has the age property >= 18, and what happens afterward, we can handle in another part of the logic flow.
Also, methods that have a return type (and not a boolean), should reflect what the method is returning. Here, at least for me `adulthoodOfPerson` is a bit confusing and if I just want to use this method in some logic, I would most probably check the implementation to be sure of the return value.